src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
```

Comment:
Perhaps put the class in a package?
See [Coding Standard - Package and Import Statements](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements).